openquake/server/templates/engine/index.html

```diff
@@ -82,9 +82,9 @@ <h3>
                 <input class="aristotle-input" type="text" id="usgs_id" name="usgs_id" placeholder="{{ aristotle_form_placeholders.usgs_id }}" value="{{ aristotle_default_usgs_id }}"/>
               </div>
               <div class="aristotle_form_row">
-                <label for"rupture_file_from_usgs">{{ aristotle_form_labels.rupture_file_from_usgs }}</label>
-                <input class="aristotle-input" type="hidden" id="rupture_file_from_usgs" name="rupture_data_file_from_usgs" placeholder="{{ aristotle_form_placeholders.rupture_file_from_usgs }}" disabled />
-                <input class="aristotle-input" type="text" id="rupture_file_from_usgs_loaded" name="rupture_file_from_usgs_loaded" disabled />
+                <label for"rupture_from_usgs">{{ aristotle_form_labels.rupture_from_usgs }}</label>
```

Comment:
The propagation of the downloaded/converted rupture to the job still needs to be fixed